### PR TITLE
Make section buttons fixed in mobile

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
Make section buttons fixed in mobile with a blurred background

<div class='graphite__hidden'>
          <div>🎥 Video uploaded on Graphite:</div>
            <a href="https://app.graphite.dev/media/video/nv8d5VeBPARA7ezh3Od3/19802524-0d03-41e9-b9e5-291f730c857a.mov">
              
            </a>
          </div>
<video src="https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/nv8d5VeBPARA7ezh3Od3/19802524-0d03-41e9-b9e5-291f730c857a.mov">Screen Recording 2024-10-09 at 11.18.54.mov</video>

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
* **#4838** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4838?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gustaveen and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/features/priceCalculator/FormGridNew.css.ts [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
+import { CONTENT_MAX_WIDTH } from './PurchaseFormV2.css'
+
+// Offset for fixed formFooter
+const GRID_PADDING_BOTTOM = '7rem'
 
 export const grid = style({
   display: 'grid',
   gridTemplateColumns: 'repeat(6, 1fr)',
   gap: tokens.space.xxs,
+  paddingBottom: GRID_PADDING_BOTTOM,
```
Comment (alebedev) [inline apps/store/src/features/priceCalculator/FormGridNew.css.ts:12]: Do we need it in wide layout?

File: apps/store/src/features/priceCalculator/FormGridNew.css.ts [hunk 2]
```diff
@@ -14,6 +19,29 @@ export const gridItem = style({
...
+  paddingInline: tokens.space.md,
+  backdropFilter: 'blur(50px)',
+
+  ...responsiveStyles({
+    lg: {
+      position: 'revert',
+      gridColumn: 'span 6',
+      width: 'auto',
+      padding: 0,
+      backdropFilter: 'revert',
```
Comment (alebedev) [inline apps/store/src/features/priceCalculator/FormGridNew.css.ts:44]: Just curious - any difference from `unset` here?

**Review by alebedev — APPROVED** [review]

**Review by guilhermespopolin — APPROVED** [review]

File: apps/store/src/features/priceCalculator/FormGridNew.css.ts (hunk 1 above)
Comment (gustaveen) [reply apps/store/src/features/priceCalculator/FormGridNew.css.ts:12]: We don't, thanks! 🙌

File: apps/store/src/features/priceCalculator/FormGridNew.css.ts (hunk 2 above)
Comment (gustaveen) [reply apps/store/src/features/priceCalculator/FormGridNew.css.ts:44]: Mm I change my mind, let's do unset. Unset fallbacks if there would be inherited values.

**Comment by gustaveen** [comment]
### Merge activity

* **Oct 10, 7:20 AM EDT**: A user merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4838).

**Pull request merged**